.github/workflows/pixel-validation.yml

```diff
@@ -0,0 +1,50 @@
+name: Pixel Schema Validation
+
+on:
+    push:
+      branches:
+      - develop
+    pull_request:
+      paths:
+      - 'PixelDefinitions/**'
+
+jobs:
+    pixel-validation:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Use Node.js 20
+              uses: actions/setup-node@v4
+              with:
+                node-version: 20.x
+
+            - uses: actions/cache@v4
+              with:
+                path: ~/.npm
+                key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+                restore-keys: |
+                  ${{ runner.os }}-node-
+
+            - name: Checkout internal-github-asana-utils repo
+              uses: actions/checkout@v4
+              with:
+                  token: ${{ secrets.DAXMOBILE_ANDROID_AUTOMATION }}
+                  path: internal-github-asana-utils
+                  repository: duckduckgo/internal-github-asana-utils
+
+            - name: Install dependencies
+              run: npm ci --verbose
+              working-directory: PixelDefinitions
+
+            - name: Lint
+              run: npm run lint || (npm run lint.fix; git diff; echo 'Pixel lint check failed, review diff above or run "npm run lint.fix"'; exit 1)
```

Comment:
I think `git diff` will hang indefinitely because it will open the pager, at least that's what I see when testing locally so would do `git --no-pager diff` instead.